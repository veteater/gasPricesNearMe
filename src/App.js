import React from 'react';

function App() {
  return (
    <div className="App">
      Gas Prices Near Me
    </div>
  );
}

export default App;
