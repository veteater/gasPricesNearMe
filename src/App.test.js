import React from 'react';
import { shallow, mount } from 'enzyme';
import App from './App';

describe('App', () => {
  it('renders without crashing', () => {
    const app = shallow(<App />);
    expect(app).toMatchSnapshot();
  });

  it('mounts will full matched snapshot', () => {
    const app = mount(<App />);
    expect(app).toMatchSnapshot();
  });
});
