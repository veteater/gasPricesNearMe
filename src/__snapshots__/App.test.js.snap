// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders the starting text 1`] = `
<div>
  <div
    class="App"
  >
    <div
      id="GasPrices"
    >
      <header>
        <h1>
          Gas Prices Near Me
        </h1>
        <article
          class="locationControls"
        >
          <span>
            Find Near Me
          </span>
          <span>
            - or -
          </span>
          <span>
            <input
              placeholder="Enter a zipcode..."
              readonly=""
              type="text"
              value="zipcode"
            />
            <span>
              Find
            </span>
          </span>
        </article>
      </header>
      <article
        class="results"
      >
        <span
          class="logo"
        >
          Logo
        </span>
        <div
          class="info"
        >
          <h2>
            title
          </h2>
          <ul>
            <li>
              Regular: $0.00
            </li>
            <li>
              Premium: $0.00
            </li>
            <li>
              Diesel: $0.00
            </li>
          </ul>
          <span>
            Directions
          </span>
        </div>
      </article>
    </div>
  </div>
</div>
`;
